docs/AxilrodTellerPotential.md

```diff
@@ -0,0 +1,35 @@
+# Axilrod-Teller(-Muto) Potential
+
+[Reference Paper](https://pubs.aip.org/aip/jcp/article/11/6/299/182086/Interaction-of-the-van-der-Waals-Type-Between)
+
+![3-body-sketch](./graphics/3_body_sketch.png)
+
+The Axilrod-Teller potential is a model for the interactions of three molecules which appear when the van der Waals forces are approximated to the third order.
+
+$$ U_{AT} = \nu \frac{3 \cos\gamma_1 \cos\gamma_2 \cos\gamma_3 + 1}{r_{12}^3 r_{23}^3 r_{31}^3} $$
```

Comment:
```suggestion
$$
U_{AT} = \nu \frac{3 \cos\gamma_1 \cos\gamma_2 \cos\gamma_3 + 1}{r_{12}^3 r_{23}^3 r_{31}^3}
$$
```

These math environments are super brittle. One space in the wrong location can break them.